articles/app-service/quickstart-php.md

```diff
@@ -18,20 +18,21 @@ zone_pivot_groups: app-service-platform-windows-linux
 ::: zone pivot="platform-linux"
 [Azure App Service](overview.md) provides a highly scalable, self-patching web hosting service.  This quickstart shows how to deploy a PHP app to Azure App Service on Linux.
 
-You create and deploy the web app using [Azure CLI](/cli/azure/get-started-with-azure-cli).
-
 ![Sample app running in Azure](media/quickstart-php/hello-world-in-browser.png)
 
 You can follow the steps here using a Mac, Windows, or Linux machine. Once the prerequisites are installed, it takes about five minutes to complete the steps.
 
 To complete this quickstart, you need:
 
 1. An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/).
-1. <a href="https://git-scm.com/" target="_blank">Git</a>
-1. <a href="https://php.net/manual/install.php" target="_blank">PHP</a>
-1. <a href="/cli/azure/install-azure-cli" target="_blank">Azure CLI</a> to run commands in any shell to provision and configure Azure resources.
+2. <a href="https://git-scm.com/" target="_blank">Git</a>
```

Comment:
I'd change all of these to bulleted list instead of numbered list.

Reply:
[Resolved](https://github.com/MicrosoftDocs/azure-docs/pull/93186/commits/544627bc723d0e9ab18e240af1563d9af05cc3e5)